packages/jupyterlab-chat-extension/src/index.ts

```diff
@@ -493,7 +501,8 @@ const chatCommands: JupyterFrontEndPlugin<void> = {
     if (launcher) {
       launcher.add({
         command: CommandIDs.createChat,
-        category: 'Other'
+        category: 'Other',
+        args: { fromLauncher: true }
```

Comment:
I think that `inSidePanel` should be set to false for the launcher case to keep the side panel UX focused on chat files in the default dir and make it work like other launcher created files.